include/ring-core/base.h

```diff
@@ -89,6 +89,14 @@
 #elif defined(__MIPSEL__) && defined(__LP64__)
 #define OPENSSL_64_BIT
 #define OPENSSL_MIPS64
+#elif defined(__riscv) && __riscv_xlen == 64
+#define OPENSSL_64_BIT
+#define OPENSSL_RISCV64
+#define OPENSSL_NO_ASM
```

Comment:
BoringSSL doesn't define OPENSSL_NO_ASM here. Is it really needed?

Reply:
Right, this is deprecated. Removed in 71dc9e9